StandardConfig/production/HighLevelReco/HighLevelReco.xml

```diff
@@ -306,43 +306,61 @@
 
 
   <!--########## add TOF estimators for 0, 10 and 50 ps resolution  ###################################### -->
-
+  ExtrapolateToEcal
+  TofMethod
+  TimeResolution
+  MaxEcalLayer
+  Verbosity
```

Comment:
Should these still be here, or are they leftovers from implementing the changes?